box_embeddings/common/constant.py

```diff
@@ -0,0 +1 @@
+TANH_EPS = 1e-20
```

Comment:
Seems like you do not use this constant in the tanh box tensor file. If so, should we drop the file for now?

Reply:
Oh nevermind, I see that you have used it.